books/models.py

```diff
@@ -849,6 +849,12 @@ def save(self, *args, **kwargs):
         if self.support_statement:
             Book.objects.filter(locale=self.locale).update(support_statement=self.support_statement)
 
+        # if book is new, clear out isbn 10 fields
+        if self._state.adding:
```

Comment:
@edwoodward what is the difference between this and setting `default=None` on the [fields](https://github.com/openstax/openstax-cms/blob/master/books/models.py#L475-L476)?

Reply:
@mwharrison When a book is translated, the data is copied over to the new book. This is to clear the fields on translated books which was the issue.